content/api/authentication/enterpriseuser/_index.en.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Autentication with enterprise users
+linktitle: Enterprise users
+description: Description how to use enterprise users in Altinn 3
+toc: true
+weight: 100
+--
+
+## Overall description
+
+Business users is a concept where you can use a business certificate in combination with a username / password.
```

Comment:
The title talk about enterprise users, but the rest of the document use business instead. I think the term _enterprise_ is best.

Reply:
Fixed